README.md

```diff
@@ -1 +1,188 @@
+}
+~~~
+
+The Spring Boot starter will also provide a `com.cloudant.client.api.Database` bean if your provide a `cloudant.db` property. For example:
+
+~~~
+cloudant.url=http://cloudant.com
+cloudant.username=myUsername
+cloudant.password=myPassword
+cloudant.db=myDb
+~~~
+
+~~~ java
+@Autowired
+private Database db;
+
+public List<Greeting> getAllDocsAsGreetings() {
+    return List<Greeting> allDocs = db.getAllDocsRequestBuilder().includeDocs(true).build().getResponse().getDocsAs(Greeting.class);
+}
+~~~
+
+To provide custom connection data you can override the `com.cloudant.client.api.ClientBuilder` bean:
+
+~~~ java
+@SpringBootApplication
+public class DemoApplication {
+
+    public static void main(String[] args) {
+        SpringApplication.run(DemoApplication.class, args);
+    }
+
+    @Bean
+    public ClientBuilder builder() {
+        ClientBuilder builder = ClientBuilder
+            .account("example")
+            .username("exampleUser")
+            .password("examplePassword");
+        return builder;
+    }
+}
+~~~
+
+### Spring Framework Applications
+
+You must provide the following properties:
+
+* cloudant.url
+* cloudant.username
+* cloudant.password
+
+To enable the creation of the `com.cloudant.client.api.CloudantClient` bean you must add an `com.cloudant.spring.framework.EnableCloudant` annotation to your application configuration:
+
+~~~ java
+@Configuration
+@EnableWebMvc
+@EnableCloudant
+@ComponentScan
+public class SpringConfig {}
+~~~
+
+~~~ java
+@Autowired
+private CloudantClient client;
+
+public List<String> getAllDbs() {
+    return client.getAllDbs();
+}
+~~~
+
+## Related documentation
+* [Cloudant library for Java](https://github.com/cloudant/java-cloudant/)
+* [Cloudant docs](https://console.bluemix.net/docs/services/Cloudant/cloudant.html#overview)
+* [Cloudant Learning Center](https://developer.ibm.com/clouddataservices/cloudant-learning-center/)
+
+# Development
+
+For information about contributing, building, and running tests see the [CONTRIBUTING.md](CONTRIBUTING.md).
+
+### Using in Other Projects
+
+The preferred approach for using cloudant-spring in other projects is to use the Gradle or Maven dependency as described above.
+
+### License
+
+Copyright © 2017 IBM Corp. All rights reserved.
+
+Licensed under the apache license, version 2.0 (the "license"); you may not use this file except in compliance with the license.  you may obtain a copy of the license at
+
+    http://www.apache.org/licenses/LICENSE-2.0.html
+
+Unless required by applicable law or agreed to in writing, software distributed under the license is distributed on an "as is" basis, without warranties or conditions of any kind, either express or implied. See the license for the specific language governing permissions and limitations under the license.
+
+### Issues
+
+Before opening a new issue please consider the following:
+* Only the latest release is supported. If at all possible please try to reproduce the issue using
```

Comment:
This is "un-supported" in the first instance so remove the first sentence here (and probably "if at all possible").